drivers/flash/soc_flash_nrf.h

```diff
@@ -8,6 +8,8 @@
 #define __SOC_FLASH_NRF_H__
 
 #include <kernel.h>
+#include <soc.h>
+#include <autoconf.h>
```

Comment:
Did you really have to include this one? I thought that was handled in a different way.
Like this in the compile command: `-imacros /home/joakim/repos/ncs/nrf/build/zephyr/include/generated/autoconf.h`

Reply:
Yes, you are right. I didn't know about this.

Reply:
Fixed